src/main/java/com/yahoo/bullet/bql/extractor/QueryExtractor.java

```diff
@@ -118,6 +123,9 @@ private Query constructQuery() {
         query.setFilters(filters);
         query.setProjection(projection);
         query.setWindow(window);
+        if (postAggregations != null && !postAggregations.isEmpty()) {
```

Comment:
Why do we only null and empty check postAggregations and not Filters, Projection or Window?